packages/client/src/util/rpc.ts

```diff
@@ -22,13 +23,13 @@ type CreateRPCServerOpts = {
   logger?: Logger
 }
 type CreateRPCServerReturn = {
-  server: RPCServer
+  server: any
```

Comment:
Can't we type those two as jayson.Server?

Reply:
Indeed we can.  Thanks for catching this!